src/createServer.js

```diff
@@ -1,3 +1,78 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { detectCase } = require('./convertToCase/detectCase');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
```

Comment:
It's recommended to use a popular web framework like Express.js for creating the server as it simplifies a lot of things like routing, request handling, etc. However, if you want to stick with the native http module, that's fine as well.